src/chart.cpp

```diff
@@ -454,7 +454,7 @@ void Chart3D::pushTicktextCoords(float x, float y, float z)
 void Chart3D::generateChartData()
 {
     CheckGL("Begin Chart3D::generateChartData");
-    static const float border[] = { -1, -1, -1,  1, -1, -1,  -1, -1, -1,  -1, 1, -1,  -1, 1, -1,  -1, 1, 1 };
+    static const float border[] = { -1, -1, 1,  -1, -1, -1,  -1, -1, -1,  1, -1, -1,  1, -1, -1,  1, 1, -1 };
```

Comment:
@syurkevi  did you change the order of vertices or something ? On a quick look, that's what it seems to me, was this the fix or it was an auxiliary change ?